test/test_flop_counter.py

```diff
@@ -0,0 +1,124 @@
+# Owner(s): ["module: unknown"]
+
+import torch
+from torch.testing._internal.common_utils import TestCase, run_tests
+import torch.utils.flop_counter
+import unittest
+
+try:
+    from torchvision import models as torchvision_models
+    HAS_TORCHVISION = True
+except ImportError:
+    HAS_TORCHVISION = False
+skipIfNoTorchVision = unittest.skipIf(not HAS_TORCHVISION, "no torchvision")
+
+def FlopCounterMode(*args, **kwargs):
+    return torch.utils.flop_counter.FlopCounterMode(*args, **kwargs, display=False)
+
+def get_total_flops(mode):
+    return str(sum([v for _, v in mode.flop_counts["Global"].items()]))
```

Comment:
It seems the test would be simpler if this function just returns int.

Reply:
assertExpectedInline expects strings.